client/src/pages/MapPage/MapViewer.vue

```diff
@@ -49,27 +49,8 @@ export default {
   },
 
   mounted() {
-    // Create map and controls
-    this.map = new MapView({
```

Comment:
Comments are good! :)

Reply:
Would you like me to add additional in this area?

Reply:
Nope - Just letting you know I found them very clarifying.